pyaccel/accelerator.py

```diff
@@ -192,7 +191,7 @@ def vchamber_on(self, value):
     @property
     def lattice_version(self):
```

Comment:
Why didn't you implement a setter for this property as well?

Reply:
I wanted this property to be rigid. My initial thought on this property was to set it as read-only, and its definition happens only at `create_accelerator` method in `pyaccel`. This string would be updated with our lattice updates in `pymodels`. Models created before this change would have an empty string in `lattice_version`. I don't see a case that it would be recommended overwriting this property once it is already defined in the accelerator class from pyaccel. Nevertheless, if you think that having a setter is important, I can implement it.

Reply:
What if we create a model from pymodels and then make some changes, like introducing an ID, or other component? 
I think we could let the user give its own interpretation to the meaning of this property and not take an authoritarian  road, don't you think?

Reply:
Yes, now I agree and I think you're right, `lattice_version` could be the original version + additional comments, intermediate version numbers, etc. I implemented the setter. Flexibility wins!